ldaptor/test/test_distinguishedname.py

```diff
@@ -154,13 +160,25 @@ class LDAPDistinguishedName_RFC2253_Examples(TestCaseWithKnownValues):
          [[('1.3.6.1.4.1.1466.0', '#04024869')],
           [('O', 'Test')],
           [('C', 'GB')]]),
+        )
 
-        (u'SN=Lu\u010di\u0107'.encode('utf-8'),
-         [[('SN', u'Lu\u010di\u0107'.encode('utf-8'))]])
```

Comment:
This is still possible, but it was moved into a dedicated tescase with a dedicated docstring see LDAPDistinguishedName_UTF8_Init